lib/tasks/proxy.rake

```diff
@@ -32,4 +37,17 @@ namespace :proxy do
   task :set_correct_endpoint_hosted => :environment do
     Service.includes(:proxy).where(proxies: {endpoint: nil}, deployment_option: 'hosted').find_each(&:deployment_option_changed)
   end
+
+  desc 'Update proxy rules owners'
+  task update_proxy_rule_owners: :environment do
```

Comment:
So I think for now it is acceptable to just document it.
Let see later if we can have a mechanism to make some post-migration automatic and resilient over uprades